ui-components/src/helpers/filters.ts

```diff
@@ -50,6 +51,7 @@ export interface ListingFilterState {
   [FrontendListingFilterStateKeys.minRent]?: string | number
   [FrontendListingFilterStateKeys.maxRent]?: string | number
   [FrontendListingFilterStateKeys.seniorHousing]?: string | boolean
+  [FrontendListingFilterStateKeys.include_nulls]?: boolean
```

Comment:
This should have type `string | boolean` since we aren't currently parsing the input from the query string into the correct type.